src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -3190,6 +3190,7 @@ namespace Microsoft.FSharp.Collections
                 let start = if i < 0 then 0 else i
                 PrivateListHelpers.sliceTake (j - start) (PrivateListHelpers.sliceSkip start l)
 
+        [<Experimental("Experimental library feature, requires '--langversion:preview'")>]
```

Comment:
I think these should all say, `Preview library feature` since we aren't really experimenting with these.

Reply:
I din't feel like inventing a whole new concept, the documentation for the experimental attribute calls it an **experimental library feature**.  It seemed more than good enough to me.
````
This attribute is used to tag values that are part of an experimental library feature.
````
https://msdn.microsoft.com/visualfsharpdocs/conceptual/core.experimentalattribute-class-%5bfsharp%5d

Reply:
Note that I'm not saying to remove the attribute, but suggesting that the message say that it is a `Preview library feature` and not an experimental one.